cpp/arcticdb/entity/type_utils.cpp

```diff
@@ -198,4 +198,16 @@ namespace arcticdb {
         }
     }
 
+    // The above method requires that both types are exactly representable by one type
```

Comment:
Can we add the name of the method. It might become confusing if code gets reordered.